addons/crm/tests/test_crm_lead_lost.py

```diff
@@ -92,7 +92,7 @@ def test_lead_lost_batch_wfeedback(self):
             self.assertEqual(lead.probability, 0)
             self.assertEqual(lead.lost_reason_id, self.lost_reason)
             # check messages
-            self.assertEqual(len(lead.message_ids), 3, 'Should have 3 messages: creation, lost, and log')
```

Comment:
Commit message says "when losting a lead" ? loosing? or marking as lost if you want to be specific.

Also: 

> from user an UI point of view

typo?